docs/content/en/docs/installation/helm/_index.md

```diff
@@ -1,35 +1,58 @@
 ---
-title: "Installing with Helm"
-linkTitle: "Installing with Helm"
-weight: 15
+title: "Installing with Helm (recommended)"
+linkTitle: "Installing with Helm (recommended)"
+weight: 10
 ---
 
 The Aiven Operator for Kubernetes can be installed via [Helm](https://helm.sh/). 
 
-First add the [aiven helm repository](https://github.com/aiven/aiven-charts):
+First add the [Aiven Helm repository](https://github.com/aiven/aiven-charts):
 
 ```bash
-helm repo add aiven https://aiven.github.io/aiven-charts && helm repo update
+$ helm repo add aiven https://aiven.github.io/aiven-charts && helm repo update
 ```
 
-Next install the custom resource definitions:
+## Installing Custom Resource Definitions
 
 ```bash
-helm install aiven-operator-crds aiven/aiven-operator-crds
+$ helm install aiven-operator-crds aiven/aiven-operator-crds
 ```
 
-Finally install the operator:
+Verify the installation
+```bash
+$ kubectl api-resources --api-group=aiven.io
+
+NAME                  SHORTNAMES   APIVERSION          NAMESPACED   KIND
+connectionpools                    aiven.io/v1alpha1   true         ConnectionPool
+databases                          aiven.io/v1alpha1   true         Database
+... < several omitted lines >
+```
+
+## Installing the Operator
 
 ```bash
-helm install aiven-operator aiven/aiven-operator
+$ helm install aiven-operator aiven/aiven-operator
 ```
 
-It is also possible to install the operator without webhooks enabled:
+> Note: Installation will fail if the CRDs for the cert-manager are not installed
 
+Verify the installation: 
 ```bash
-helm install aiven-operator aiven/aiven-operator --set webhooks.enabled=false
+$  helm status aiven-operator
+
+NAME: aiven-operator
+LAST DEPLOYED: Fri Sep 10 15:23:26 2021
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
 ```
 
+> It is also possible to install the operator without webhooks enabled:
```

Comment:
Since this became a part of its own (not a special paragraph) I'd remove the `>` chars.

Reply:
:heavy_check_mark: